shopfloor/actions/message.py

```diff
@@ -952,3 +983,13 @@ def lot_change_no_line_found(self):
             "message_type": "error",
             "body": _("Unable to find a line with the same product but different lot."),
         }
+
+    def picking_type_is_return(self, picking):
```

Comment:
The fact that it is a return is a specific use case. Make the method name more generic.
```suggestion
    def reserved_for_other_picking_type(self, picking):
```